packages/block-editor/src/components/collab/block-comment-menu-item.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+import { _x } from '@wordpress/i18n';
+import { MenuItem } from '@wordpress/components';
+import { collabComment } from '@wordpress/icons';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { store as blockEditorStore } from '../../store';
+
+export default function BlockCommentMenuItem( { onClose } ) {
+	// eslint-disable-next-line @wordpress/data-no-store-string-literals
+	const { openGeneralSidebar } = useDispatch( 'core/edit-post' );
+
+	// eslint-disable-next-line @wordpress/data-no-store-string-literals
+	const { updateBlockAttributes } = useDispatch( blockEditorStore );
+
+	const clientId = useSelect( ( select ) => {
+		// eslint-disable-next-line @wordpress/data-no-store-string-literals
+		const { getSelectedBlockClientId } = select( blockEditorStore );
+		return getSelectedBlockClientId();
+	}, [] );
+
+	const openCollabBoard = () => {
+		onClose();
+		updateBlockAttributes( clientId, {
+			showCommentBoard: true,
+		} );
+		openGeneralSidebar( 'edit-post/collab-sidebar' );
+	};
+
+	return (
+		<MenuItem
+			icon={ collabComment }
+			onClick={ openCollabBoard }
+			aria-haspopup="dialog"
+		>
+			{ _x(
+				'Comment',
+				'Click to open new comment form in comment sidebar'
```

Comment:
Context is usually in passive language and short. Did you mean to add a translators comment instead? For example:

```
/* Translators: Text of the button to add a new comment in the comment sidebar. */
__( 'Comment' )
```

If you really meant to add context, I'd recommend:

```
_x( 'Comment', 'Add comment button' )
```

or:
```
_x( 'Comment', 'verb' )
```


Note that this applies to multiple instances in this PR.